app/assets/stylesheets/overrides.scss

```diff
@@ -55,15 +55,8 @@
   margin-top: 1em;
 }
 
-.input-group--inline .form-group {
-  display: inline-block;
-  margin-bottom: 0;
-}
-
-.range-text {
-  margin: 0 1em;
-}
-
-.form-divider {
-  margin: 1em;
+.main-footer {
```

Comment:
This is two changes in one: 1) moving non-override CSS to custom.scss, and 2) upping the contrast on our footer links to make a11y specs pass.